apps/src/componentLibrary/link/Link.tsx

```diff
@@ -31,12 +31,14 @@ export interface LinkBaseProps extends HTMLAttributes<HTMLAnchorElement> {
 export type LinkWithChildren = LinkBaseProps & {
   /** Link content */
   children: React.ReactNode;
+  className?: string;
```

Comment:
@bencodeorg LinkBaseProps already have optional classname prop. Why do we need this here?
Also, if adding anything to DSCO components - you need to update CHANGELOG.md for that component

Reply:
🤦 My bad, misread the type definition. Fixed here!

https://github.com/code-dot-org/code-dot-org/pull/61915